packages/generators/init-template/webpack-template/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "webpack-template",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {},
```

Add a `prepare` script to automatically enable Git hooks.
```suggestion
  "scripts": {
    "prepare": "husky install"
  },
```